### ISSUE TITLE
Magnific Popup Conflict with Feed Them Social

### ISSUE THREAD

**Issue by mathetos** [WordImpress/Give#1273]
## Issue Overview

The "Feed Them Social" (FTS) plugin also uses Magnific Popup in their plugin and they also enqueue the JS globally. This means that if a Give user is using the modal popup display method, the FTS popup ALSO pops up and actually has a higher z-index than ours which makes our form unusable.

## Expected Behavior
Plugins that also use Magnific like we do, shouldn't get triggered from our modal buttons.

## Possible Solution
Override the default `magnificPopup` function as described here:
http://dimsemenov.com/plugins/magnific-popup/documentation.html#faq

## Steps to Reproduce (for bugs)
1. Install Give
2. Install Feed Them Social
3. Create a Give form that uses the Modal display method
4. Go to the new form and click the Donate Now button and you'll see results similar to this:

![image](https://cloud.githubusercontent.com/assets/930724/20500707/6f80ec86-afeb-11e6-995c-4510ddb1715e.png)

**Comment by mehul0810** [comment]
@mattimpress I'm unable to reproduce this issue. I've tried it by installing latest Give and Feed Them Social and then saved settings of Feed Them Social and created a shortcode of FTS plugin. Now, I've created a Give Form with modal display settings and then tried opening modal and it worked fine. Also, put FTS shortcode and Give Form shortcode in a page and checked it again. Still worked fine. 

- Tried in incognito mode and works fine.
- Tried with Twenty Seventeen and Twenty Fourteen and works fine.

Here is a GIF with FTS Plugin enabled:
![gif](http://g.recordit.co/yOSFKlQYoR.gif)
